dexs/zkera-finance/zkera.ts

```diff
@@ -0,0 +1,45 @@
+import { request, gql } from "graphql-request";
+import { FetchOptions } from "../../adapters/types";
+import BigNumber from "bignumber.js";
+
+const graphUrl =
+  "https://api.goldsky.com/api/public/project_clzelu5d634f501x8ai8111wj/subgraphs/zlp-pool/latest/gn";
+const getData = async (timestamp: number) => {
+  const query = gql`
+    {
+      volumeStats(first: 1, where: { period: daily, timestamp: ${timestamp} }) {
```

Comment:
we count `
      liquidation
      margin` for volume could you fix ?

Reply:
also, how does this timestamp field work, it returns data for 24 hours before given timestamp or after?

Reply:
> we count ` liquidation margin` for volume could you fix ?

What do you mean? Can you explain?

Reply:
> this example https://github.com/DefiLlama/dimension-adapters/blob/master/dexs/gmx/index.ts#L22

volumeStats(where: { period: daily, timestamp: ${timestamp}}, first: 1) {
    timestamp
    mint
    burn
    margin
    swap
    liquidation
  }


there is separate field for liquidations in the graph